ArduPlane/quadplane.cpp

```diff
@@ -2995,7 +3004,7 @@ bool QuadPlane::land_detector(uint32_t timeout_ms)
     // we only consider the vehicle landed when the motors have been
     // at minimum for timeout_ms+1000 and the vertical position estimate has not
     // changed by more than 20cm for timeout_ms
-    if (fabsf(height - landing_detect.vpos_start_m) > 0.2) {
+    if (fabsf(height - landing_detect.vpos_start_m) > land_detect_threshold) {
```

Comment:
This resets `land_start_ms` which is then compared against the timeout passed in. That is hard codded to 4 seconds, I wonder if we would be better to make that timeout a parameter rather than this distance. The timeout is use for both the lower motor limit check and this movement check. So that timeout would give more control.

Reply:
i prefer to leave the motor limit checks as 4s. I think adjusting height change threshold is the easiest way to handle the issue, outside of EKF changes